sections/1-introduction.tex

```diff
@@ -1,101 +1,76 @@
+The mission statement of the \sunpyproj is to facilitate and promote the use and development of several community-led, free, and open-source\footnote{\url{https://opensource.org/osd}} data analysis software packages based on the scientific \python\footnote{\url{https://www.python.org/}} environment.
+To achieve this goal, the project develops and maintains a core package (\sunpypkg) and supports an ecosystem of affiliated packages (see Section \ref{sec:affil_packages}) that provides additional functionality. 
+The project was formally founded in March of 2014 and released version 0.5 three months later \citep{Community:2015cy}, although development of the core package began in 2011. 
 
+The \sunpyproj selected the \python programming language to leverage the rich ecosystem of packages already available for general data analysis. 
+These include \package{Numpy} for multi-dimensional array manipulation \citep{numpy}, \package{scipy} for scientific functions \citep{scipy}, \package{Matplotlib} for publication-quality 2D plotting \citep{matplotlib}, and \package{Pandas} for data structures and time series analysis \citep{pandas}.
+These core packages form the backbone for hundreds of thousands of additional \python packages.
+Of particular relevance to \sunpypkg is the \astropypkg package, which provides core functionality for the analysis of astrophysical data \citep{astropy2018}. 
 
-
-%NumPy 102906 LOC
-%SciPy 139009 LOC
-%matplotlib 98720 LOC
-%pandas 212338 LOC
-%astropy 151825 LOC
-
-%The SunPy Project aims to develop and provide high-quality, maintainable and tested code paired with extensive documentation that follow current best practices in software development. 
-
-%The core library sets a standard and example for other codes to follow.
-
-%A description of the current affiliated packages can be found in Section~\ref{sec:affil_packages}.
-
-\section{Project Organization \& Enhancement Proposals - Steven Christe}
-% edited by Steven C. 23-Apr-2019
+\section{Project Organization \& Enhancement Proposals}
 
 The organization of the \sunpyproj is modeled on the structure of a board-only not-for-profit corporate entity. 
 It consists of an up-to 10 member self-selected board. 
-An executive director, elected by the board, is the lead developer whose responsibilities include leading the developer community, providing user support, developing and maintaining the core package, as well as supporting the development of affiliated packages. 
-The lead developer is supported by a deputy as well as other volunteers from the developer community. 
+An executive director, elected by the board, leads the development of \sunpy core package, provides user support, and supports the development of affiliated packages. 
+As such, the executive director is also the \sunpy lead developer.
+A deputy lead developer and release manager, as well as other volunteers from the developer community, support the lead developer.
```

Comment:
maybe add, "and form the core development team". The term "core team" is frequently by Python.